dxtbx/tests/tst_imageset.py

```diff
@@ -659,6 +659,43 @@ def run(self):
 
     print 'OK'
 
+class Test_SACLA_MPCCD_Cheetah_File(object):
+
+  def __init__(self):
+    from os.path import join
+    self.filename = join(dials_regression,
+                    "./image_examples/SACLA_MPCCD_Cheetah/run266702-0-subset.h5")
+
+  def run(self, just_in_time=False):
+
+    from dxtbx.format.Registry import Registry
+    format_class = Registry.find(self.filename)
+
+    iset = format_class.get_imageset([self.filename], just_in_time=just_in_time)
+
+    assert len(iset) == 4
+    for i in range(len(iset)):
+      data = iset.get_raw_data(i)
+#      mask = iset.get_mask(i)
+      b = iset.get_beam(i)
```

Comment:
if there is nothing else to be done with ```b``` then change to
```assert iset.get_beam(i)```
to guarantee a truthy return value. Same for the following 7 assignments